src/ui/components/Header/ViewToggle.tsx

```diff
@@ -78,7 +54,7 @@ function ViewToggle({ viewMode, setViewMode }: PropsFromRedux) {
 
 const connector = connect(
   (state: UIState) => ({
-    viewMode: getViewMode(state),
+    toggleMode: getToggleMode(state),
```

Comment:
`ToggleMode`'s kind of confusing without any context. Maybe `ViewToggleMode`? Even then I'm not a fan of the double verb, but at least it's somewhat connected. I'll leave it to you.